spec/jobs/dhis2/ethiopia_exporter_job_spec.rb

```diff
@@ -62,4 +88,94 @@
       end
     end
   end
+
+  describe "#format_gender_age_data" do
+    subject { described_class.new.send(:format_gender_age_data, gender_age_data) }
+
+    let(:gender_age_data) { {["female", 33] => 74, ["male", 96] => 72, ["female", 31] => 4, ["male", 8] => 6} }
+    it "returns the data in gender and age range key" do
+      output = subject
+      expect(output.count).to eq 8
+      expect(output["18_29_male"]).to eq 0
```

Comment:
[question] What happens if we receive data outside the given ranges, like say, when there are patients that are under the age of 18? Are we supposed to discard it? If yes, can we include that case in the test?